docs/cloud/aws/lca1/billing/budget-actions.md

```diff
@@ -0,0 +1,96 @@
+
+    Next to the amount, choose **Absolute value** to be notiﬁed when your costs exceed the threshold amount. Or, choose **% of budgeted amount** to be notiﬁed when your costs exceed the threshold percentage.
+
+    Next to the threshold, choose **Actual** to create an alert for actual spend "Trigger". Or, choose **Forecasted** to create an alert for forecasted spend.
+
+   ```{image} images/5.png
+   :alt: AWS Budget creation wizard, setting alert notifications
+   ```
+
+3. (Optional) Under **Notification preferences - Optional**, for **Email recipients**, enter the email addresses that you want the alert to notify. Separate multiple email addresses with commas. A notification can have up to 10 email addresses.
+
+4. (Optional) Under **Notification preferences - Optional**, for **Amazon SNS Alerts**, enter the Amazon Resource Name (ARN) for your Amazon SNS topic. For instructions on how to create a topic, see [Creating an Amazon SNS topic for budget notifications](https://docs.aws.amazon.com/cost-management/latest/userguide/create-cost-budget.html).
+
+5. (Optional) Under **Notification preferences - Optional**, for **AWS Chatbot Alerts**, you can configure AWS Chatbot to send budget alerts to an Amazon Chime or Slack chat room. You configure these alerts through the AWS Chatbot console.
+
+6. You may choose to add more alert thresholds for various actual amounts or various % of budgeted amounts. For instance, one alert for 80% and another for 100% of budgeted amount. Once configured, Choose Next.
+
+7. For **Attach actions - Optional**, choose **Add Action.**
+
+   ```{image} images/6.png
+   :alt: AWS Budget creation wizard, attaching actions
+   ```
+
+8. For **Select IAM role**, choose an IAM role to allow AWS Budgets to perform an action on your behalf. It is recommended to use a role which has the Managed policy: `AWSBudgetsActions_RolePolicyForResourceAdministrationWithSSM` attached to it. This managed policy is focused on specific actions that AWS Budgets takes on your behalf when completing a specific action. This policy gives permission to control AWS resources. For example, starts and stops Amazon EC2 or Amazon RDS instances by running AWS Systems Manager (SSM) scripts.
+
+9. For **Which action type should be applied when the budget threshold has been exceeded**, select the action that you want AWS Budgets to take on your behalf. You can choose from applying an IAM policy or targeting specific Amazon resources. You can apply multiple budget actions to a single alert.
+
+10. If you want to choose "Automate instance to stop for EC2 or RDS", choose related fields to select "region" and "instances" that you intend to stop.
+
+11. For **Do you want to automatically run this action when this threshold is exceeded**, choose **Yes** or **No**. If you choose No, then you run the action manually on the **Alert details page**. For instructions, see Reviewing and approving your budget action.
```

Comment:
Is this supposed to be a link?

Reply:
Good catch. Fixed in 728cd298e752e0ade98095f7403aa5e23ba5f305.